docs/source/index.rst

```diff
@@ -42,3 +42,10 @@ Documentation
 
 .. _Package Reference: p3.html
 
+Contribute
+##########
+
+Contributions to the P3 Analysis Library are welcome in the form of issues and
+pull requests. Please see the `GitHub repository`_ for more information.
+
+.. _GitHub repository: https://github.com/intel/p3-analysis-library
```

Comment:
Do we want to just link to `CONTRIBUTING.md` directly?

Reply:
Good idea.  Fixed in e32bde4e764e5ad09dfa79da21ac0cf211db4d36.